src/main/java/org/folio/rest/client/FeeFinePubSubClient.java

```diff
@@ -0,0 +1,71 @@
+package org.folio.rest.client;
+
+import static io.vertx.ext.web.client.WebClient.create;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.HttpStatus;
+import org.folio.rest.exception.InternalServerError;
+import org.folio.rest.jaxrs.model.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpResponse;
+
+public class FeeFinePubSubClient {
```

Comment:
Looks like this class and `PubSubService` are doing the same thing. Should we merge them together?

Reply:
For the `account with loan closed event` we need to forward failures to the end-user in case event publish was rejected.

However, pub/sub rejects event publish with code 400 in case there are no subscribers for this event. By introducing this client, we're overriding this behavior, so it is treated as success. In few words, it is  an equivalent of the `PubSubClientUtils.sendEventMessage` method from the `pub-sub-client` lib. I'd keep it as separate class.

What do you think?

Reply:
Thanks for pointing this 400 error out, I did not consider this case. Handling the error for missing subscribers correctly is a good idea. But you also made me think if we should fail the whole operation if event publishing failed in other cases too...

I will resolve this comment.